internal/docker/docker_test.go

```diff
@@ -25,14 +25,15 @@ type PassFailCase struct {
 	Name           string
 	Client         CommonAPIClient
 	JobConfig      *config.Project
+	Suite		   *config.Suite
```

Comment:
@alexplischke weird identation

Reply:
Indeed! It's not in the actual file though 🤔